File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x<0:
            return False
        if x==0:
            return True
        numbers = []
        flg = True
        while(x):
            numbers.append(x%10)
            x=x//10
        for i in range(len(numbers)):
            if(numbers[i]!=numbers[len(numbers)-i-1]):
                return False
        return flg

File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x<0) return false;
        string num = to_string(x);
        int l =0, r = num.size()-1;
        while(l<r){
            if(num[l]!=num[r]) return false;
            l++;
            r--;
        }
        return true;
    }
};